src/Mod/TechDraw/Gui/CommandAnnotate.cpp

```diff
@@ -147,24 +147,24 @@ bool CmdTechDrawLeaderLine::isActive(void)
 }
 
 //===========================================================================
-// TechDraw_RichAnno
+// TechDraw_RichTextAnnotation
 //===========================================================================
 
-DEF_STD_CMD_A(CmdTechDrawRichAnno)
+DEF_STD_CMD_A(CmdTechDrawRichTextAnnotation)
 
-CmdTechDrawRichAnno::CmdTechDrawRichAnno()
-  : Command("TechDraw_RichAnno")
+CmdTechDrawRichTextAnnotation::CmdTechDrawRichTextAnnotation()
+  : Command("TechDraw_RichTextAnnotation")
 {
     sAppModule      = "TechDraw";
     sGroup          = QT_TR_NOOP("TechDraw");
-    sMenuText       = QT_TR_NOOP("Add a rich text annotation");
-    sToolTipText    = QT_TR_NOOP("Add a rich text annotation");
-    sWhatsThis      = "TechDraw_RichAnno";
+    sMenuText       = QT_TR_NOOP("Insert Rich Text Annotation");
+    sToolTipText    = sMenuText;
```

Comment:
I'm not sure I like this style. If you see how Draft does it, the menu item is just the name of the tool, for example, "Polygon". The "Insert" or "Draw" action is implied: "Insert Polygon" or "Draw Polygon".

I also don't like "Title Style" in which the Initial Letter of a Word is Capitalized. I think this is ugly in general. "Sentence style" would be my preference; only the first letter is capitalized.

Original: "Add a rich text annotation"
Your proposal: "Insert Rich Text Annotation"
My proposal: "Rich text annotation"

And also, the `sToolTipText` should not be the same as `sMenuText`. The tooltip is a place where you can write a bit of information on how to use the tool. It serves as a mini documentation, so the user doesn't have to consult the entire documentation (wiki). A complete sentence that can be translated is ideal here.

```cpp
sToolTipText = QT_TR_NOOP("Insert a block of text of multiple lines and paragraphs "
                          "where the text can be formatted in different styles; "
                          "bold, italics, justification, and custom sizes is supported");
```